src/js/views/classSession.js

```diff
@@ -278,6 +283,31 @@ export default class ClassSession extends Component {
     );
   }
 
+  renderCalories() {
```

Comment:
move to separate component that has both calories and heart rate. call it something generic that indicates it's feedback/metrics for the user